README.md

```diff
@@ -668,6 +668,20 @@ npm run build -- --with-netlify
 netlify deploy
 ```
 
+### AWS Lambda (experimental)
+
+```
+npm run build -- --with-aws-lambda
+```
+
+The generated output format is compatible with [AWS Lambda](https://aws.amazon.com/lambda/).
+The output entrypoint `dist/serve.js` is compatible with [AWS Lambda format](https://docs.aws.amazon.com/lex/latest/dg/lambda-input-response-format.html).
+
+You can zip and upload the `dist` folder [manual to AWS console](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-package.html) or use one of the tools to deploy:
+
+- [serverless framework](https://www.serverless.com)
+- [AWS CDK](https://aws.amazon.com/de/cdk/)
+
```

Comment:
I would like to keep README simple, like just a few lines as others.
It's like to demonstrate that it is possible.

If we need some detailed instructions, my suggestion is to add `./docs/*.mdx`, and add a link in README.

```suggestion
You can zip and upload the `dist` folder [manual to AWS console](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-package.html) with the entrypoint `dist/serve.js`.

```

Reply:
I understand but the I would suggest to only point to the [Hono AWS Lambda Deploy Docs] - have look on my latest version.